apps/src/studio/studio.js

```diff
@@ -2027,6 +2067,10 @@ Studio.setSprite = function (opts) {
   // Reset height and width:
   sprite.height = sprite.size * skin.spriteHeight;
   sprite.width = sprite.size * skin.spriteWidth;
+  if (skin.projectileSpriteHeight)
```

Comment:
If that's our chosen style, we should totally enable [JSHint's 'curly' option](http://jshint.com/docs/options/#curly) globally.